src/lib/storage/chunk.hpp

```diff
@@ -162,6 +176,8 @@ class Chunk : private Noncopyable {
   pmr_vector<std::shared_ptr<BaseIndex>> _indices;
   std::shared_ptr<ChunkStatistics> _statistics;
   bool _is_mutable = true;
+  mutable uint64_t _invalid_row_count = 0;
+  CommitID _cleanup_commit_id = MvccData::MAX_COMMIT_ID;
```

Comment:
I've said it elsewhere, but this is probably the right place: this would probably better be a std::optional